src/pybind/cephfs/cephfs.pyx

```diff
@@ -177,7 +201,15 @@ cdef extern from "cephfs/libcephfs.h" nogil:
     int ceph_fsync(ceph_mount_info *cmount, int fd, int syncdataonly)
     int ceph_conf_parse_argv(ceph_mount_info *cmount, int argc, const char **argv)
     int ceph_chmod(ceph_mount_info *cmount, const char *path, mode_t mode)
+    int ceph_fchmod(ceph_mount_info *cmount, int fd, mode_t mode)
     int ceph_chown(ceph_mount_info *cmount, const char *path, int uid, int gid)
+    int ceph_fchown(ceph_mount_info *cmount, int fd, int uid, int gid)
+    int ceph_lchown(ceph_mount_info *cmount, const char *path, int uid, int gid)
+    int ceph_utime(ceph_mount_info *cmount, const char *path, utimbuf *buf);
+    int ceph_futime(ceph_mount_info *cmount, int fd, utimbuf *buf);
+    int ceph_utimes(ceph_mount_info *cmount, const char *path, timeval times[2])
+    int ceph_lutimes(ceph_mount_info *cmount, const char *path, timeval times[2]);
+    int ceph_futimes(ceph_mount_info *cmount, int fd, timeval times[2]);
```

Comment:
As Ramana mentioned, some of these are already implemented in the PR I linked in the tracker ticket: https://tracker.ceph.com/issues/44171

Reply:
Yes, I will remove those from my PR and update.